src/main/kotlin/com/workos/WorkOS.kt

```diff
@@ -156,6 +165,19 @@ class WorkOS(
     return sendRequest(buildRequest(request, config), responseType)
   }
 
+  /**
+   * Performs a POST request with WorkOS configuration parameters.
+   */
+  fun post(path: String, config: RequestConfig? = null) {
```

Comment:
Added an overload to the `post` method for endpoints in which we want to return a void response.